portal/src/locale-data/en.json

```diff
@@ -136,60 +137,20 @@
   "CreateAppScreen.error.invalid-app-id": "Invalid Project ID. Project ID should start and end with lowercase alphanumeric characters or numbers, and it should contain only lowercase alphanumeric characters, numbers or '-'",
```

Comment:
Better to rename `CreateAppScreen` to `CreateProjectScreen`, and remove the unused translations.